pandas/plotting/_core.py

```diff
@@ -1278,6 +1278,9 @@ def hist(self, by=None, bins=10, **kwargs):
         ----------
         by : str or sequence, optional
             Column in the DataFrame to group by.
+
+            .. versionadded:: 1.3.0
```

Comment:
Besides the release note, this should also be updated to the new expected release version.

Reply:
yeah, changed! 

One thing to note: I also updated this for `box` plot, since `by` argument is there for `box` already, but just not being implemented (before this PR, we don't do anything with `by`). So here instead of `versionadded`, I use `versionchanged` for `box` plot, and still use `versionadded` for `hist` since there was no `by` argument.